bodhi/client/__init__.py

```diff
@@ -1060,12 +1060,17 @@ def info_release(name, url, **kwargs):
               help='Display full list, including archived releases.')
 @url_option
 @debug_option
+@add_options(pagination_options)
 @staging_option
-def list_releases(display_archived, url, **kwargs):
+def list_releases(display_archived, url, rows=None, page=None, **kwargs):
     """Retrieve and print list of releases."""
+    if not display_archived and rows is not None:
+        click.echo("ERROR: must set --display_archived to use pagination")
+        sys.exit(1)
```

Comment:
I'm not sure we should block the user from doing this. Sure, it's unnecessary to page when there are so few options, but I don't think we need to stop them from doing it.

Reply:
bodhi cli omits archived releases if flag is display_archived is disabled, so when we type for instance  rows=2 and page=1, nothing is printed. The reason is that filtering releases state is done after request is  received(in cli side). So should I remove mentioned guard?

Reply:
Ah, I see now. I hadn't realized the archived filtering happens client side. That is pretty strange. It seems like this whole thing would work better if that flag just queried the server for releases based on whether they are archived or not, rather than filtering them client side.